project.local.json

```diff
@@ -17,7 +17,7 @@
       }
     },
     "Db": {
-      "cryptKey": "<crypt-key>",
+      "cryptKey": "a31e5e809da3ea6544925ffe630c5932",
```

Comment:
I think you can keep the placeholder because this key can change from a project to another.